src/patterns/components/pagination/default.hbs

```diff
@@ -0,0 +1,45 @@
+---
+name: Default
+notes: |
+  This is the pattern for when you have three or fewer pages. All three page numbers and prev/next
+  links that appear in the pagination are clickable. The current page number should be
+  bold and have a background color. When the Next link is tapped or clicked the next sequential
+  page is shown and the background color should be removed from the current page number
+  and shown on the next sequential page number. The opposite
+  is true for the Prev link. If the first or last page are selected then the previous
+  or next links should be disabled respectively.
+order: 2
+---
+<nav aria-label="Pagination Navigation" data-sprk-pagination="default">
+  <ul class="sprk-c-Pagination sprk-b-List sprk-b-List--bare">
```

Comment:
Shouldn't pagination be an ordered list?

Reply:
No, because Prev and Next aren't numbers.